package.json

```
@@ -2,11 +2,11 @@
   "name": "jsonld-request",
```

This should become a scoped package and the old package should be deprecated and point at the new scoped one.

```suggestion
  "name": "@digitalbazaar/jsonld-request",
```

---

Hmm. I did consider this, I figured you might ask, and here are thoughts I had:
- I think the only user of this package is `jsonld-cli`?  I had almost considered just merging this code back in there and deprecating this totally, for simplicity.  But I suppose this *could* be used for something else.
- I'd rather *not* scope `jsonld-cli` since it allows nice short usage like `npx jsonld-cli ...`.  And I'd rather *not* scope `jsonld` since it's nice and short and popular.  So just one scoped package in this group seems odd.
- I kind of hope one day to obsolete this package in favor of some improved system. Unknown timeline of course. Trying to avoid filling namespace with new packages that have some expectation of going away.

Do any of those thoughts change your request?

---

@davidlehn, not really. All of those packages should become scoped. We can do it later though. I won't hold up this change on account of it -- but we want to stop using non-scoped packages, it's a maintenance / authz headache.